packages/components/tsconfig.json

```diff
@@ -44,7 +44,8 @@
 		"src/h-stack/**/*",
 		"src/v-stack/**/*",
 		"src/z-stack/**/*",
-		"src/view/**/*"
+		"src/view/**/*",
+		"src/scrollable/**/*"
```

Comment:
I initially added this line in alphabetical order, but then noticed that the list is sorted until `src/visually-hidden/**/*` — and then I noticed that the "sorted" part of the list represents components that are marked as production-read, while the second (non-sorted) part is for experimental components.

Probably the best person to answer about this would be @sarayourfriend , but maybe we keep this change as-is for now and if needed sort this list alphabetically in one of the future PRs?

Reply:
Good point — I tried to give an explanation [here](https://github.com/WordPress/gutenberg/pull/32446/files/f5a7f960d041850950b461f9487c9e9df9906111#r646411977)

Reply:
Oh, that's a good point 👍  Thanks for elaborating

Reply:
Hmmm, I don't think any of that is intentional. Sorting them alphabetically is fine with me. If we want to make a separation between the experimental and prod components we should add a comment explaining that and add some spacing to make it more obvious.

Though TBH I don't think it's worth the effort to maintain two separate lists. I'd rather we just opened a PR like Marco suggested to sort them once and for all.

Reply:
Just wanted to stress that this is totally a minor deal. Maybe we can sort these when we touch the list the next time, but there's definitely no need to make a special PR just to sort them. It's why I prefaced my first comment with "nit: " 😉